crates/robonode-server/src/logic/op_authenticate.rs

```diff
@@ -109,6 +112,8 @@ where
             .await
             .map_err(Error::InternalErrorEnrollment)?;
 
+        let scan_result_blob = enroll_res.scan_result_blob.clone();
```

Comment:
Why clone?

Reply:
`enroll_res` has dropped before returning the result.

Reply:
Well, exactly - so why not deconstruct it into pieces? Take the `scan_result_blob` right from there.

Reply:
Note that even without manual destructuring, if you were to just from the clone here rust is smart enough to figure out what to do here and take the `scan_result_blob` out without cloning.

Reply:
Got it, improved - https://github.com/humanode-network/humanode/pull/1026/commits/78dd8e438226aa986178e235f8af4e83894b023d.